### PR TITLE
Snapshots from 2017-02-20

### PR DESCRIPTION
**Pull request by rbukovansky**

### CODE REVIEW

File: versions/3.1-DEVELOPMENT-SNAPSHOT-2017-02-20-a.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+binaries:
+  osx: https://swift.org/builds/swift-3.1-branch/xcode/swift-3.1-DEVELOPMENT-SNAPSHOT-2017-02-20-a/swift-3.1-DEVELOPMENT-SNAPSHOT-2017-02-20-a-osx.pkg
+  ubuntu14.04: https://swift.org/builds/swift-3.1-branch/ubuntu1610/swift-3.1-DEVELOPMENT-SNAPSHOT-2017-02-20-a/swift-3.1-DEVELOPMENT-SNAPSHOT-2017-02-20-a-ubuntu16.10.tar.gz
```
Comment (kylef) [inline versions/3.1-DEVELOPMENT-SNAPSHOT-2017-02-20-a.yaml]: This is the URL for `ubuntu16.10`.

File: versions/DEVELOPMENT-SNAPSHOT-2017-02-20-a.yaml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+binaries:
+  osx: https://swift.org/builds/development/xcode/swift-DEVELOPMENT-SNAPSHOT-2017-02-20-a/swift-DEVELOPMENT-SNAPSHOT-2017-02-20-a-osx.pkg
+  ubuntu14.04: https://swift.org/builds/development/ubuntu1404/swift-DEVELOPMENT-SNAPSHOT-2017-02-20-a/swift-DEVELOPMENT-SNAPSHOT-2017-02-20-a-ubuntu14.04.tar.gz
+  ubuntu16.04: https://swift.org/builds/development/ubuntu1604/swift-DEVELOPMENT-SNAPSHOT-2017-02-20-a/swift-DEVELOPMENT-SNAPSHOT-2017-02-20-a-ubuntu16.04.tar.gz
+  ubuntu16.10: https://swift.org/builds/development/ubuntu1404/swift-DEVELOPMENT-SNAPSHOT-2017-02-20-a/swift-DEVELOPMENT-SNAPSHOT-2017-02-20-a-ubuntu14.04.tar.gz
```
Comment (kylef) [inline versions/DEVELOPMENT-SNAPSHOT-2017-02-20-a.yaml]: This is the URL for `ubuntu14.04`.

Comment (rbukovansky) [inline versions/DEVELOPMENT-SNAPSHOT-2017-02-20-a.yaml]: Sorry, my bad! Corrected in https://github.com/kylef/swiftenv-api/pull/26/commits/ec34014b48a26f32bc189f7978810eed798b341e

**Comment by rbukovansky** [comment]
@kylef Reworked, sqaushed commits. Now it should be all good.

Thanks for review.

**Comment by rbukovansky** [comment]
@kylef @ikesyo Any takers?

**Pull request merged**